src/commands/deployWorkspaceProject/deploymentConfiguration/getTreeItemDeploymentConfiguration.ts

```diff
@@ -10,6 +10,7 @@ import { localize } from "../../../utils/localize";
 import { type DeploymentConfiguration } from "./DeploymentConfiguration";
 
 export async function getTreeItemDeploymentConfiguration(item: ContainerAppItem | ManagedEnvironmentItem): Promise<DeploymentConfiguration> {
+    // Todo: Search and add container registry
```

Comment:
Was testing something and noticed this needed to be an additional todo, although not directly related to this PR